static/app/gettingStartedDocs/javascript/astro.tsx

```diff
@@ -0,0 +1,201 @@
+import {Fragment} from 'react';
+
+import ExternalLink from 'sentry/components/links/externalLink';
+import {StepType} from 'sentry/components/onboarding/gettingStartedDoc/step';
+import {
+  BasePlatformOptions,
+  Docs,
+  DocsParams,
+  OnboardingConfig,
+} from 'sentry/components/onboarding/gettingStartedDoc/types';
+import {t, tct} from 'sentry/locale';
+// import {getPackageVersion} from 'sentry/utils/gettingStartedDocs/getPackageVersion';
+
+const platformOptions = {} satisfies BasePlatformOptions;
+
+type PlatformOptions = typeof platformOptions;
+type Params = DocsParams<PlatformOptions>;
```

Comment:
`DocsParams` and the other interfaces have a default, so you can simply omit it. 
```suggestion
type Params = DocsParams;
```